### ISSUE TITLE
please create a record for St. Appian of Caesarea

### ISSUE THREAD

**Issue by jsaintlauren** [srophe/srophe-app-data#106]
https://archive.org/stream/bub_gb_7p2iW6B-15IC#page/n243/mode/2up

**Comment by wlpotter** [comment]
I don't think this record was created? (It's not showing up in a simple keyword search in SBD). @nathangibson @jsaintlauren do you know if this was resolved?

**Comment by nathangibson** [comment]
The above link says the martyrology is taken from Assemani II, 189–195. Thus it is "Apphianus" https://syriaca.org/person/1609, commemorated in https://syriaca.org/work/911, which cites the same text. 

Thus we only need to add any suitable names, not create a new record. @jsaintlauren it looks like you're suggesting "St. Appian of Caesarea"?

**Comment by jsaintlauren** [comment]
Record needs to include martyr element and more robust description.

**Comment by wlpotter** [comment]
Okay, so this has turned out to be a bit more complicated...

Apphianus (https://syriaca.org/person/1609) is commemorated in https://syriaca.org/work/911. This is BHO no. 95. Work 911 has Apphianus as a martyr under Maximinus (I believe Maximinus II, according to Assemani's death date of 306 -- https://archive.org/details/ActaSanctorumMartyrumOrientaliumEtOccidentalium2/page/n193/mode/2up).

According to BHO p. 23 (https://archive.org/details/BibliothecaHagiographicaOrientalis/page/n45/mode/1up), this person is the same as Epiphanius commemorated by Eusebius in the _Martyrs of Palestine_.

This same person is commemorated, according to the BHO entry (no. 94) in http://syriaca.org/work/236. We have work 236 commemorating Epiphanius, http://syriaca.org/person/1708.

Are https://syriaca.org/person/1609 and http://syriaca.org/person/1708 the same person?
